src/Sarif/Autogenerated/SarifRewritingVisitor.cs

```diff
@@ -544,6 +544,18 @@ public virtual Resources VisitResources(Resources node)
         {
             if (node != null)
             {
+                if (node.Rules != null)
+                {
+                    var keys = node.Rules.Keys.ToArray();
+                    foreach (var key in keys)
+                    {
+                        var value = node.Rules[key];
+                        if (value != null)
+                        {
+                            node.Rules[key] = VisitNullChecked(value);
+                        }
+                    }
+                }
```

Comment:
>} [](start = 16, length = 1)

Here, after all the shouting, is the correctly generated dictionary visitation code for `resources.rules`. `run.graphs` and `result.graphs` are visited below.